frontend/plan/pages/index.tsx

```diff
@@ -142,120 +142,154 @@ function Index() {
+                        mobileView={true}
+                        storeLoaded={storeLoaded}
+                        isExpanded={isExpanded}
                     />
-                    <Tab
-                        className="topTab"
-                        label="Schedule"
-                        onClick={() => setTab(2)}
+                    <Tabs value={tab} className="topTabs" centered>
+                        <Tab
+                            className="topTab"
+                            label="Search"
+                            onClick={() => setTab(0)}
+                        />
+                        <Tab
+                            className="topTab"
+                            label="Cart"
+                            onClick={() => setTab(1)}
+                        />
+                        <Tab
+                            className="topTab"
+                            label="Schedule"
+                            onClick={() => setTab(2)}
+                        />
+                    </Tabs>
+                    <SwipeableViews
+                        index={tab}
+                        // @ts-ignore
+                        ref={containerRef}
+                        enableMouseEvents
+                        onSwitching={scrollTop}
+                        onChangeIndex={setTab}
+                    >
+                        <div
+                            style={{
+                                paddingLeft: "10px",
+                                paddingRight: "10px",
+                            }}
+                        >
+                            <div>
+                                <div
+                                    style={{
+                                        display: "flex",
+                                        flexDirection: "row",
+                                        justifyContent: "space-around",
+                                        margin: "10px",
+                                    }}
+                                >
+                                    <SearchSortDropdown />
+                                </div>
+                                <div
+                                    className="box"
+                                    style={{
+                                        paddingLeft: 0,
+                                        paddingRight: 0,
+                                    }}
+                                >
+                                    <Selector mobileView={true} view={0} />
+                                </div>
+                            </div>
+                        </div>
+                        <div style={{ padding: "10px" }}>
+                            <Cart setTab={setTab} mobileView={true} />
+                        </div>
+                        <div style={{ padding: "10px" }}>
+                            {/* Unfortunately running into a weird issue with connected components and TS. */}
```

Comment:
Out of curiosity, what was the specific issue here?

Reply:
honestly, no clue. Was saying a prop that was defined on the interface, wasn't actually defined. think it has to do with `connect` but can't be 100% sure